tests/test_core.py

```diff
@@ -1168,3 +1171,83 @@ def test_has_single_liaisons_true():
     liaisons = [0, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0]
     output = has_single_liaisons(liaisons)
     assert output
+
+
+def test_remove_exact_length_matches():
+    lines = [
+        {'tokens': [
+            {'word': [
+                {'syllable': 'me',
+                 'is_stressed': False,
+                 'has_synalepha': True,
+                 'is_word_end': True}],
+                'stress_position': 0},
+            {'word': [
+                {'syllable': 'he', 'is_stressed': True, 'is_word_end': True}],
+                'stress_position': -1}],
+            'phonological_groups': [
+                {'syllable': 'mehe',
+                 'is_stressed': True,
+                 'synalepha_index': [1],
+                 'is_word_end': True}],
+            'rhythm': {'stress': '+-',
+                       'type': 'pattern',
+                       'length': 2,
+                       'length_range': {'min_length': 2, 'max_length': 3}}},
+        {'tokens': [{'word': [
+            {'syllable': 'i', 'is_stressed': True},
+            {'syllable': 'do', 'is_stressed': False, 'is_word_end': True}],
+            'stress_position': -2}],
+            'phonological_groups': [
+                {'syllable': 'i', 'is_stressed': True},
+                {'syllable': 'do', 'is_stressed': False, 'is_word_end': True}],
+            'rhythm': {'stress': '+-',
+                       'type': 'pattern',
+                       'length': 2,
+                       'length_range': {'min_length': 2, 'max_length': 2}}}]
```

Comment:
This should not happen (length is 2 always, so length_range is not needed)

Reply:
It's part of the `test_remove_exact_length_matches` so the input should have that and the output should have it removed, I think it's fine

Reply:
Damn, sorry for skimming over it. :+1: